src/resdk/collection_tables.py

```diff
@@ -263,7 +268,18 @@ def _load_fetch(self, data_type: str) -> pd.DataFrame:
             if data_type == META:
                 data = self._download_metadata()
             else:
-                data = self._download_expressions(data_type)
+                try:
+                    loop = asyncio.new_event_loop()
+                    asyncio.set_event_loop(loop)
```

Comment:
You should almost never replace an existing event loop use [`get_event_loop`](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_event_loop) instead.